### ISSUE TITLE
Favicon

### ISSUE THREAD

**Issue by ploeh** [ploeh/ploeh.github.com#75]
The blog should have a favicon...

**Comment by JoeMighty** [comment]
How about one of your face? I'll make one for you :)

**Comment by ploeh** [comment]
Heh, isn't that a little too self-centred?

**Comment by JoeMighty** [comment]
I guess some might see it that way. Though I would argue that some people (including myself in this instance) come to your website because they value your opinion. So therefore you're the brand that your website is promoting. And Scott Hanselman has his face as an favicon! ;)

If not your face then a small P for Ploeh?

**Comment by ploeh** [comment]
If you take the letters from *ploeh* and squash them all together, and then subsequently ignore the *h*, you get something like this:

![ploeh glyph](https://cloud.githubusercontent.com/assets/242165/5852447/27da08f6-a217-11e4-87c5-c9e423abfc42.jpg)

If you then tilt it 45°, it fits pretty well within a square:

![ploeh glyph tilted](https://cloud.githubusercontent.com/assets/242165/5852452/47fb18dc-a217-11e4-8633-9ac6887a4de8.jpg)

That's just about as obscure as the word *ploeh* in itself. Would that work?

**Comment by feO2x** [comment]
What is the meaing of "Ploeh"? It is not a danish word as far as I can tell. If it has a special meaning, this could be incorporated into your favicon.

**Comment by ploeh** [comment]
I could tell you, but then I'd have to kill you ;)

**Comment by feO2x** [comment]
Ok, let's leave it at that :-)

**Comment by feO2x** [comment]
Maybe something like this?

![ploeh favicon](https://cloud.githubusercontent.com/assets/7884158/5853370/56ff675a-a223-11e4-9c92-85e2bbaf4a69.png)

**Comment by ploeh** [comment]
Considering that the most common size users are going to see is a 16x16 pixel image, I'm not too concerned about how the eye scans the icon.

My thought with the above was to have the line go from corner to corner; otherwise, I'm afraid it'd look weird. However, I do grant that it may look strange with half of the icon being empty, so perhaps it's better just to scrap that entire concept...

**Comment by feO2x** [comment]
I wouldn't necessarily say so:
![ploeh favicon v0 2](https://cloud.githubusercontent.com/assets/7884158/5855316/34465a3e-a237-11e4-8a29-3f0c22f537fd.png)

This is how it looks in 16x16:
![ploeh favicon v0 2 16x16](https://cloud.githubusercontent.com/assets/7884158/5855318/39f62a54-a237-11e4-9653-48947f15424b.png)

The letters 'p' and 'h' might not be as distinguishable as before, but it's a logo - there should be some creative freedom.

Please tell me if I should create a Pull Request for you and if I should include the Adobe Illustrator file with it, too.

**Comment by ploeh** [comment]
Sorry to appear ungrateful, but that doesn't really work for me. Thank you for the attempt, though :)

**Comment by feO2x** [comment]
Next try - what do you think about this?
![ploeh favicon](https://cloud.githubusercontent.com/assets/7884158/6044147/e10e096a-ac91-11e4-8a66-9d620a8d4b0b.png)

In 16x16 and 32x32 they would look like this:
![ploeh favicon 16x16](https://cloud.githubusercontent.com/assets/7884158/6044163/f299d984-ac91-11e4-8419-523b4ad463dc.png)    
![ploeh favicon 32x32](https://cloud.githubusercontent.com/assets/7884158/6044167/f9555df2-ac91-11e4-999f-963a84c8fc98.png)

Basically I took the initials of "ploeh blog" and applied some figure-ground motive.

**Comment by ploeh** [comment]
Looks great :+1: 

I'd very much like to use this icon. Would you like to send a 'real' pull request?

**Comment by feO2x** [comment]
Should I include only the favicon or the Adobe Illustrator file, too?

**Comment by ploeh** [comment]
It's up to you. The website only needs the icon, and I don't have Illustrator, so personally, I have no need for it.

**Comment by ploeh** [comment]
Done in 642652d51aeac047d09dd34acd2b631f0337bcb5
